uploadNeuroDB/register_processed_data.pl

```diff
@@ -142,7 +151,7 @@
     my  $psc    =   $center_name;
     if  (!$psc)     { 
         print LOG "\nERROR: No center found for this candidate \n\n"; 
-        exit 77; 
+        exit $NeuroDB::ExitCodes::GET_PSC_FROM_SOURCEFILEID_FAILURE;
```

Comment:
why not use the already defined `$GET_PSC_FAILURE` defined as exit code 91 (from MRIProcessingUtility.pm)? it seems to be the same error to me; don't you think?

Reply:
Good point. I was actually on the fence about this one when I created it but could not decide ;). Will be using $GET_PSC_FAILURE